### DISCUSSION TITLE
Custom message on blocked/delayed page

### DISCUSSION THREAD

**Discussion by Kenqr** [proginosko/LeechBlockNG#470] [Ideas]
I would like to show a customized message on the blocked/delayed page, telling me why I should take a break from these sites and what alternative things I can do.

**Comment by shiny-ribbon** [comment]
I see after posting that you made a PR adding the option. So this should be solved for OP.

But for anyone else who might be reading and looking for another or simpler way:
If you're willing to learn a little bit of HTML, it is easy to do with an HTML page. I'll share my HTML page here with whoever wants it. Once you've downloaded it, right-click and edit it with Notepad. From there it's up to you what you want to do but there's a lot of different things you can put, change the text, put links, and do some time math. You just have to read through the HTML and try to make sense of it. Easier said than done but it's doable haha
https://drive.google.com/file/d/1XuLPywz8kjc-LtKkN_sLR7WXOrUz1gn2/view?usp=sharing

Then just double-click to open the file in your browser and take the link from the address bar and put it in Leechblock custom URL field

**Comment by proginosko** [comment]
Added in version 1.6.6.
